### DISCUSSION TITLE
Scala Days 2023 (Madrid): Storch presentation by Sören Brunk is available online

### DISCUSSION THREAD

**Discussion by hmf** [bytedeco/storch#74] [General]
Just to let everyone know. Here is the link:

[Sören Brunk - Storch - GPU Accelerated Deep Learning for Scala 3](https://www.youtube.com/watch?v=ZoJanBS9kw8&list=PLLMLOC3WM2r6ZFhFfVH74W-sl8LfWtOEc&index=19&pp=iAQB)

**Comment by sbrunk** [comment]
Thanks @hmf. I've also added the video to the website now: https://storch.dev/
